facia/public/humans.txt

```diff
@@ -195,3 +195,8 @@ Location: London, UK
 
 Developer: Jamie Byers
 Location: London, UK
+
```

Comment:
Hey, I'm in alumni as well!
This needs rearranging, alumni and colleagues is surely meant to be for people who've left.